aquadoggo/src/db/stores/query.rs

```diff
@@ -954,7 +986,17 @@ impl SqlStore {
         "#
         );
 
-        let mut rows: Vec<QueryRow> = query_as::<_, QueryRow>(&sea_quel)
+        println!("{sea_quel}");
+
+        let mut query = query_as::<_, QueryRow>(&sea_quel);
+
+        for value in bind_parameters {
+            let value = value_sql(&value);
+            println!("Bind: {}", value);
+            query = query.bind(value);
+        }
+
+        let mut rows: Vec<QueryRow> = query
```

Comment:
Ugh, I thought this would work and be a nice solution to sanitising user inputs without disturbing the current code too much. But it's not working at all, seems like the placeholder params are being composed correctly (`$1` in the right place) and `bind()` is being called for all values, but then... no documents come back from the query :thinking: hoping I'm just missing something simple...

Reply:
Any thoughts @adzialocha ? :pray:

Reply:
Could we use `push` or `push_unseparated` to build (unfinished) SQL fragments instead of passing over the `bind_parameters` for each method call and assembling it all in the end? Think that might be easier to wrap your head around / maybe easier to not forget one parameter etc.?